--- a/setup.py	2019-12-19 01:51:16.000000000 +0100
+++ b/setup.py	2019-12-20 02:03:57.192722700 +0100
@@ -105,7 +105,7 @@
             elif sys.version_info[0:2] in ((3, 3), (3, 4)):
                 toolset = ' toolset=msvc-10.0'
             elif sys.version_info[0:2] in ((3, 5), (3, 6)):
-                toolset = ' toolset=msvc-14'
+                toolset = ' toolset=msvc-14.00'
             else:
                 # unknown python version, lets hope the user has the right version of msvc configured
                 toolset = ' toolset=msvc'
@@ -121,7 +121,7 @@
         os.environ['LIBTORRENT_PYTHON_INTERPRETER'] = '"' + sys.executable + '"'
 
         # build libtorrent using bjam and build the installer with distutils
-        cmdline = ('b2 libtorrent-link=static boost-link=static release '
+        cmdline = ('b2 crypto=openssl libtorrent-link=static boost-link=static release '
                    'optimization=space stage_module --abbreviate-paths' +
                    address_model + toolset + parallel_builds)
         print(cmdline)
